infrastructure/deployment/deploy.sh

```diff
@@ -366,6 +372,7 @@ echo
 echo "Waiting 2 mins for mongo to deploy before working with data. Please note it can take up to 10 minutes for the entire stack to deploy in some scenarios."
 echo
 
+sleep 120 # Required as Kibana cannot be immediately contacted
 echo "Setting up Kibana config & alerts"
 
 while true; do
```

Comment:
This while loop should run until the curl commands succeeds. Did the commands fail silently? I originally removed the 120s sleep as in some cases it wasn't enough to guarantee ES + Kibana was up and didn't handle for random failures.

Reply:
The issue was that on a brand new server alert rules were not added into Kibana because Kibana wasnt ready to receive them.